src/Sylius/Bundle/UserBundle/Authentication/DefaultAuthenticationSuccessHandler.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\UserBundle\Authentication;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Http\Authentication\DefaultAuthenticationSuccessHandler as SymfonyDefaultAuthenticationSuccessHandler;
+use Symfony\Component\Security\Http\ParameterBagUtils;
+use Symfony\Component\Security\Http\Util\TargetPathTrait;
+
+/**
+ * TODO: Workaround for regression introduced in Symfony 3.3. To be deleted when fixed.
+ *
+ * @see https://github.com/symfony/symfony/pull/23411
+ * @see https://github.com/symfony/symfony/pull/23061
+ *
+ * @internal
+ *
+ * {@inheritdoc}
+ */
+class DefaultAuthenticationSuccessHandler extends SymfonyDefaultAuthenticationSuccessHandler
```

Comment:
It's extended by `Sylius\Bundle\UserBundle\Authentication\AuthenticationSuccessHandler` (and internal :)).